### DISCUSSION TITLE
以巴衝突：美國一票否決聯合國加沙停火決議案

### DISCUSSION THREAD

**Discussion by chu109502015** [1101-GS4719-C/Python-Community-GS4719-C#1773] [General]
在上星期五，也就是12/8，聯合國安理會呼籲立即在加沙實行人道主義停火的決議草案被美國一票否決。
以巴衝突一開始時，我以為不會打很久，因為以色列應該可以快速地打敗哈瑪斯，但沒想到以色列是想要推平加砂，其他國家也沒想到，後來發現事情不對，聯合國便跳出來了。
之前聯合國大會在10/27號以 120 :14 的票數通過停火，但是還是沒什麼屁用，要安理會通過了才有用，而安理會的決議過不了就是過不了，因為美國有一票否決權，陷入了死循環。
但剛剛聯合國大會又通過了停火決議，以 153:10通過，而且連加拿大都贊成停火，那希望美國也可以在安理會投同意吧。
大家怎麼看以巴衝突的，歡迎討論。

https://www.bbc.com/zhongwen/trad/world-67668847
https://udn.com/news/story/6809/7637072

**Comment by Chengbosco** [comment]
感覺靠著美國撐腰，以色列就能往死裡打了欸。
這個提案比較像是說:好了你們打夠了，可以回家了。但美國這個反對票就像把瘋狗的鎖鏈解開，希望在輿論下可以快點通過，不然感覺很麻煩。
